internal/httputils/utils.go

```diff
@@ -0,0 +1,68 @@
+// Copyright IBM Corp. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package httputils
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/IBM-Blockchain/bcdb-server/pkg/types"
+)
+
+func JsonMarshalOrPanic(response interface{}) []byte {
+	bytes, err := json.Marshal(response)
+	if err != nil {
+		panic(err)
+	}
+
+	return bytes
+}
+
+// SendHTTPResponse writes HTTP response back including HTTP code number and encode payload
```

Comment:
I don't think we encode whole payload anymore, so encode may go

Reply:
Not sure I understand what you mean here?

Reply:
In comment `encode payload` - we don't encode payloads anymore

Reply:
But we do, using json.Marshal... mainly used for errors though... `payload` is the parameter name